stock_partner_version/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
It is recommended to use https://github.com/OCA/maintainer-tools#readme-generator to generate README from fragments